specify/specify7
# Add SpecifyUser to Query Combo Box in Export Feed

Added a `typeSearch` to allow searching. Users will have to configure a formatter for SpecifyUser to see anything other than  `<Formatter not defined>`

Testing instructions
0. Make sure you have a formatter defined for SpecifyUser table. Add one if not.
1. Go to ExportFeed resource in xml-editor.
2. Go to `Run as user` or `Send completion notification to user` field (which is query combo box)
3. Make sure you are able to search for different users in those combo boxes
![image](https://github.com/specify/specify7/assets/61122018/fbd38afa-a575-4332-a059-ee3b995085bc)

It is rare that databases will have a formatter for specifyUser (but that's easy to add with xml-editor, so I think it is fine). Until https://github.com/specify/specify7/pull/4312 is merged, SpecifyUser won't be visible in table list for visual editor. So, the following can be added to record object formatters to add a basic formatter for SpecifyUser
```
<format
    name="SpecifyUser"
    title="SpecifyUser"
    class="edu.ku.brc.specify.datamodel.SpecifyUser"
    default="true"
    >
    <switch single="true">
        <fields>
            <field>name</field>
        </fields>
    </switch>
</format>

```
